modules/sdk-core/test/unit/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -30,12 +30,42 @@ describe('ecdsa tss', function () {
     paillierKeyStub.onCall(1).returns(Promise.resolve(paillierKeyPairs[1]));
     paillierKeyStub.onCall(2).returns(Promise.resolve(paillierKeyPairs[2]));
 
+    const [uyShare1, uyShare2, uyShare3] = [
+      ecdsa.keyShareStep1(1, 2, 3),
+      ecdsa.keyShareStep1(2, 2, 3),
+      ecdsa.keyShareStep1(3, 2, 3),
+    ];
+
+    // Exchange commitments of y_i (uyShare1.comDecomY.commitment, uyShare2.comDecomY.commitment, uyShare3.comDecomY.commitment).
+    // Make sure each party getting commitments from all other parties before proceeding to next step.
+    const yCommitments1 = {};
+    yCommitments1[2] = uyShare2.comDecomY.commitment;
+    yCommitments1[3] = uyShare3.comDecomY.commitment;
```

Comment:
Nit:

```suggestion
    const yCommitments1 = {
    	2: uyShare2.comDecomY.commitment.
    	3: uyShare3.comDecomY.commitment,
    };
```

Same comment below.

Reply:
Done